corehq/apps/cleanup/management/commands/couch_integrity.py

```diff
@@ -1,61 +1,165 @@
 import json
 import requests
+from optparse import make_option
 from collections import defaultdict
 from django.core.management.base import LabelCommand
+from django.conf import settings
 from jsonobject import JsonObject, StringProperty, ListProperty, base
 
+# This command relies on a properly formatted json spec in order to run.
+# Here is an example of a spec:
+#
+# {
+#     "couches": [ // Couches defines the couchdb configuration you want.
+#         {
+#             "host": "commcarehq.cloudant.com",
+#             "user": "<username>", // optional, falls back to your `settings.py` file for the apprioriate user
+#             "password": "<password>", // optional, falls back to your `settings.py` file for apprioriate user
+#             "headers": {} // Optional headers to pass into the request to couch
+#         }
+#
+#     ],
+#     "suites": [ // suites defines what views you'd like to run on your couch environments
+#         {
+#             "database": "commcarehq" // choose a database to run the view on
+#             "views": [ // list of views to run on the database
+#                 "_all_docs"
+#             ]
+#         }
+#
+#     ]
+# }
+#
+# When no user or password is supplied, this command will fallback to the `settings.py`
+# file to look for the user and password. An example user defined in the `settings.py` file would look like
+# this:
+#
+# CI_COMMCAREHQ_CLOUDANT_COM_USER = 'commcarehq'
+#
+# A few things are happening here. First, you must prefix the variable with CI_. Next you have to enter the
+# host name. Since you cannot declare variables with '.' in them, use '_' instead. Lastly, you need to add the
+# suffix _USER or _PASSWORD so the command knows whether you are defining the user or the password.
+# A complete example:
+#
+# CI_COMMCAREHQ_CLOUDANT_COM_USER = 'commcarehq'
+# CI_COMMCAREHQ_CLOUDANT_COM_PASSWORD = '***'
+#
+# CI_DIMAGI003_CLOUDANT_COM_USER = 'commcarehq'
+# CI_DIMAGI003_CLOUDANT_COM_PASSWORD = '***'
+
 
 class Command(LabelCommand):
     help = ("Run couch views on docs from different database instances."
             "See couch_integrity/basic.json for example config")
     label = "config file"
 
+    option_list = LabelCommand.option_list + (
+        make_option(
+            '--wiggle',
+            dest='wiggle',
+            default=0,
+            type="int",
+            help='Define how much doc counts can be off by'
+        ),
+    )
+
     def handle_label(self, *labels, **options):
+        wiggle = options.get('wiggle', 0)
```

Comment:
not actionable, but I think `optparse` guarantees that the option will be available, so you can just do `options['wiggle']`